TKGame/Animations/ScreenTransistion.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Microsoft.VisualBasic.FileIO;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using StbImageSharp;
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Data;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace TKGame.Animations
+{
+    public class ScreenTransition
+    {
+        #region Member Data
+        private Rectangle[] sourceRectangles;
+
+        private static readonly int screenWidth = 1600;
+        private static readonly int screenHeight = 900;
+        private int spriteX = 509;
```

Comment:
These are very specific, how come?

Reply:
Don't know why, but the artist who made the sprite sheet made these neon green borders around the individual images that I had to remove and that was the dimensions of the actual image inside the borders.